packages/extension/__tests__/Home.js

```diff
@@ -118,6 +120,20 @@ it('should open search bar when clicking on the button', () => {
   expect(wrapper.find('.content__header .search').element).toBeTruthy();
 });
 
+it('should close search bar when switching to bookmarks', () => {
+  const wrapper = mount(DaHome, { store, localVue });
+  wrapper.find('.search-btn').trigger('click');
+  feed.state.showBookmarks = true;
+  expect(wrapper.find('.content__header .search').element).toBeUndefined();
```

Comment:
I believe it makes sense to check that after clicking on the button the search element is defined.
If we don't check it, it might be that it was always undefined and nothing actually changed.

Reply:
Totally agree